src/algorithmForNN/pointCloudUtils.py

```diff
@@ -151,6 +151,48 @@ def get_area_of_plane(points: o3d.geometry.PointCloud, plane_model: list) -> flo
     return current_list[1] * current_list[2]
 
 
+def get_calibration_matrix_from_calib_file(path_to_calibration_file: str) -> np.ndarray:
+    calib_matrix = np.eye(4)
+    with open(path_to_calibration_file) as file:
```

Comment:
Looks like this code reused twice, let's create io function related to kitti dataset that constructs 4x4 transformation matrix from kitti lines.